Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -35,54 +34,31 @@ interface LGPageProps extends RouteComponentProps<{}> {
 const LGPage: React.FC<LGPageProps> = props => {
   const { state } = useContext(StoreContext);
   const { lgFiles, dialogs } = state;
+  const [selectedDialogItem, setSelectedDialogItem] = useState('common');
```

Comment:
it's not true with the initial state `common`, if I visit the url like:
`http://localhost:3000/language-generation/Main`, 
`Main` is current active dialog id. suggest:
1. useState('common')  --> useState( ...  init with path .... )
2. change naming from `selectedDialogItem` to `dialogId`;

ps: I prefer compute dialogId instead of useState
```
const dialogId  = useMemo( ..., [ path, dialogs ] )
```

Reply:
why not use fileId directly?